scripts/hostcfgd

```diff
@@ -1796,5 +1797,9 @@ def main():
     daemon.start()
 
 if __name__ == "__main__":
-    main()
+    try:
+        main()
+    except Exception as e:
```

Comment:
>except [](http://example.com/codeflow?start=4&length=6)

This bugfix is not related to your main topic, we can move it to another PR. And this is not the ideal way to fix a generic issue relating systemd handle simple service return code. 
Not sure if this help [https://unix.stackexchange.com/a/734184](https://unix.stackexchange.com/a/734184)

Reply:
Fixed, change reverted